src/components/editor/property/LiteralTypeLabel.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+const LiteralTypeLabel = ({ propertyTemplate }) => {
+  const labelText = (propertyTemplate) => {
+    if (propertyTemplate.validationRegex) {
+      if (propertyTemplate.validationDataType) {
+        return `Enter a "${propertyTemplate.validationDataType}" in the form "${propertyTemplate.validationRegex}"`
+      }
+      return `Enter a literal in the form "${propertyTemplate.validationRegex}"`
+    }
+    if (propertyTemplate.validationDataType) {
+      return `Enter a ${propertyTemplate.validationDataType}`
+    }
+    return "Enter a literal"
+  }
```

Comment:
1. is there a better way of determining the  literal type other than checking the "validationDataType" ?

2. the validation data type is stored as a URI and this is what is displayed to the user ... is there a better label to show?  eg. as in https://github.com/LD4P/sinopia_editor/blob/main/static/literalDataType.json    if so, how to access that label?